src/main/kotlin/net/atos/zac/app/task/TaskRestService.kt

```diff
@@ -230,12 +230,12 @@ class TaskRestService @Inject constructor(
         }
         taakVariabelenService.setTaskData(updatedTask, restTask.taakdata)
         taakVariabelenService.setTaskinformation(updatedTask, restTask.taakinformatie)
-        flowableTaskService.completeTask(updatedTask).let {
+        val completedTask = flowableTaskService.completeTask(updatedTask).also {
             indexeerService.addOrUpdateZaak(restTask.zaakUuid, false)
             eventingService.send(ScreenEventType.TAAK.updated(it))
             eventingService.send(ScreenEventType.ZAAK_TAKEN.updated(restTask.zaakUuid))
-            return restTaskConverter.convert(it)
         }
+        return restTaskConverter.convert(completedTask)
```

Comment:
If you like, this could be chained into a `.let(restTaskConverter::convert)` and returned immediately...